dragon/runtime_docs.rb

```diff
@@ -278,11 +278,11 @@ def tick args
 - ~:down_or_held~
 - ~:up~
 ** ~args.outputs~
-~args.outputs.PROPERTY~ is how you render to the screen.
+~args.outputs.PROPERTY~ is how you render to the screen and play sounds. Rendering to the screen happens in the following order: ~solids < sprites < primitives < labels < lines < borders < debug~ and ~static_*~ are drawn above their non-static counterparts.
 *** ~.background_color~
 Set ~args.outputs.background_color~ to an ~Array~ with ~RGB~ values (eg. ~[255, 255, 255]~ for the color white).
 *** ~.sounds~
-Send a file path to this collection to play a sound. The sound file must be under the ~mygame~ directory. Example: ~args.outputs.sounds << "sounds/jump.wav"~.
+Send a file path to this collection to play a sound. The sound file must be under your game directory (for example, the ~mygame~ directory). Example: ~args.outputs.sounds << "sounds/jump.wav"~.
```

Comment:
For beginners this might be confusing, it's best to mention **mygame** directory as the basic directory always?

Reply:
Maybe by saying `(by default the ~mygame~ directory)` instead of `(for example ...)`?